cp1404practicals/Prac_05/emails.py

```diff
@@ -0,0 +1,25 @@
+def main():
+    """Create dictionary of emails and names."""
+
+    emails_and_names = {}
+    get_email = input("Email: ")
+    while get_email != "":
+        name = get_name_from_email(get_email)
+        confirmation = input("Is this your name {}? (Y/N)".format(name))
+        if confirmation.upper() != "y" and confirmation.upper() != "":
```

Comment:
We can put the .upper() at the end of the input function, this saves you using it multiple times throughout the code.
Example: confirmation = input("Is this your name {}? (Y/N)".format(name)).upper()